packages/babel-helpers/src/helpers/checkInRHS.js

```diff
@@ -0,0 +1,9 @@
+/* @minVersion 7.20.1 */
+
+export default function _checkInRHS(value) {
+  var type = typeof value;
+  if (type !== "object" && type !== "function") {
+    throw TypeError("right-hand side of 'in' should be an object, got " + type);
+  }
```

Comment:
```suggestion
  if (Object(value) !== value) {
    throw TypeError("right-hand side of 'in' should be an object, got " + typeof type);
```

This is a bit shorter.